extensions/dicom-p10-downloader/src/commandsModule.js

```diff
@@ -0,0 +1,47 @@
+import OHIF from '@ohif/core';
+import {
+  getDicomWebClientFromContext,
+  getSOPInstanceReferenceFromActiveViewport,
+} from './utils';
+import downloadAndZip from './downloadAndZip';
+
+export function getCommands(context) {
+  const actions = {
+    downloadAndZipSeriesOnActiveViewport({ servers, viewports, progress }) {
+      const dicomWebClient = getDicomWebClientFromContext(context, servers);
+      if (dicomWebClient) {
+        const reference = getSOPInstanceReferenceFromActiveViewport(viewports);
+        if (reference) {
+          downloadAndZip(dicomWebClient, reference, { progress })
+            .then(url => {
+              OHIF.log.info('Zip file successfully created:', url);
+              location.href = url;
```

Comment:
Is this going to redirect the current tab? We don't want that, it might disrupt their work. Better to force a download. This is how we do it with the SaveAs tool in Cornerstone: https://github.com/cornerstonejs/cornerstoneTools/blob/master/src/util/SaveAs.js#L34

Reply:
Nope. This is a Blob URL that directly initiates the download and leaves the browsing session as is. But this was temporary and I have already changed this in order to give the file a name.